haskell-src/Concordium/Types/Queries.hs

```diff
@@ -119,7 +119,17 @@ data ConsensusStatus = ConsensusStatus
       -- Initially this is equal to 'csGenesisBlock'.
       csCurrentEraGenesisBlock :: !BlockHash,
       -- |Time when the current era started.
-      csCurrentEraGenesisTime :: !UTCTime
+      csCurrentEraGenesisTime :: !UTCTime,
+      -- |The current duration to wait before a round times out. Present from protocol version 6.
+      csCurrentTimeoutDuration :: !(Maybe Duration),
+      -- |The current round. Present from protocol version 6.
+      csCurrentRound :: !(Maybe Round),
+      -- |The current epoch. Present from protocol version 6.
+      csCurrentEpoch :: !(Maybe Epoch),
+      -- |The trigger block time of the seedstate of the last finalized block. The first block in
+      -- the epoch with timestamp at least this is considered to be the trigger block for the epoch
+      -- transition. Present from protocol version 6.
+      csTriggerBlockTime :: !(Maybe Timestamp)
```

Comment:
Looking at this, it would be more consistent in JSON if this was UTCTime. It would be the same as the other timestamps we have here.

Reply:
I can see this is fixed due to #395. So resolving.